src/components/MenuItem.css

```diff
@@ -5,3 +5,14 @@
   margin: 1.25rem;
   text-align: left;
 }
+
+.menu__buttons {
+  border: none;
+  background-color: white;
+  padding: 0 0.5rem 0 0.5rem;
+  font-size: 25px;
```

Comment:
You could use rem as you did with the padding instead of px.

Reply:
I just was recommended that em is actually better for padding. :)